RPIPs/RPIP-committee_stipends.md

```diff
@@ -0,0 +1,130 @@
+
+The committee stipend budget to be split amongst the committee members each month is defined by the following formula.
+```math
+$$
+
+Committee Monthly Stipend Budget = TargetHours * TargetCount * TargetRate
+
+$$
+```
+
+#### Parameter Changes
+
+* The Target Hours input for a given committee MAY be adjusted via the internal vote of that committee. 
+  * Increases to Target Hours MUST have a unanimous positive vote outcome, which MUST be posted publicly.
+  * Decreases to Target Hours MUST have a majority positive vote outcome, which MUST be posted publicly. 
+  * Any Target Hours change is subject to pDAO challenge, and SHALL NOT be acted upon until 1 calendar month has passed after the decision has been announced.
+* The Target Member Count SHALL be modified only by the pDAO via a voted modification to this RPIP and SHOULD follow the usual processes for such a modification. 
+* The Global Stipend Rate SHALL be modified only by the pDAO via a voted modification to this RPIP and SHOULD follow the usual processes for such a modification. 
+
+### Current and Historic Parameter Values
+
+| Date                       | Global Stipend Rate |
+|----------------------------|---------------------|
+| 2024-03-04 (RPIP Authored) | 30 USD / hr         |
+
+#### Grants Management Committee
+
+| Date                       | Target Member Count | Target Member Hours | Total Monthly Stipend Budget |
+|----------------------------|---------------------|---------------------|------------------------------|
+| 2024-03-04 (RPIP Authored) |                   9 |                  13 |                    $3,510.00 |
+
+#### Incentives Management Committee
+
+| Date                       | Target Member Count | Target Member Hours | Total Monthly Stipend Budget |
+|----------------------------|---------------------|---------------------|------------------------------|
+| 2024-03-04 (RPIP Authored) |                   9 |                  13 |                    $3,510.00 |
+
+
+<!--
+Commentary for Editors:
+The above tables may be generated using this [google sheet](https://docs.google.com/spreadsheets/d/1aXE-bbVK5IB54TfqRPq9_kyMKMDDDiVDuw3yA3f5XJg/edit#gid=958081945), which should be maintained in parallel. If you need to modify the sheet and do not have permission then:
+1. Copy the sheet
+2. Set it to public read-only
+3. Replace this link with your updated sheet
+4. Convert the new sheet to markdown using: https://www.tablesgenerator.com/markdown_tables or similar.
+-->
+
+### Payments
+
+#### Split
+The split of the committee stipend budget SHOULD be determined by the voting members of the committee in question. 
+* Each committee SHOULD be consistent in the method used to determine the payout split. 
+* Roles that are compensated through other mechanisms SHOULD be discounted when the committee determines the stipend split.
```

Comment:
And the target member hours?

Reply:
I don't see that covered in the section above. Eg, if we pay out to "Treasurer" in a new way, that work should no longer be included when calculating TargetHours.

Reply:
Right, I misunderstood you initially. Agree, I'll make the change.